lmfdb/abvar/fq/isog_class.py

```diff
@@ -231,6 +236,20 @@ def properties(self):
             ("Geometrically simple", "yes" if self.is_geometrically_simple else "no"),
             ("Primitive", "yes" if self.is_primitive else "no"),
         ]
+        # Cyclicity information (new)
+        ic = getattr(self, "is_cyclic", None)
+        if ic is True:
+            cyclic_val = "yes"
+        elif ic is False:
+            cyclic_val = "no"
+        else:
+            cyclic_val = "unknown"
+        props.append(("Cyclic group of points", cyclic_val))
```

Comment:
Instead of adding these to the properties box, you should add them to the list of invariants in the main page.  In general, everything in the properties box should be echoed on the main page.  We also choose to only show a subset of the data in the properties box, and I don't think these two pieces of information are central enough to include.